models/DataObject/AbstractObject.php

```diff
@@ -1327,4 +1337,22 @@ public function set($fieldName, $value, $language = null)
     {
         return $this->{'set'.ucfirst($fieldName)}($value, $language);
     }
+
+    /**
+     * @return bool
+     */
+    public static function isDirtyDetectionDisabled()
+    {
+        return self::$disableDirtyDetection;
+    }
+
+    /**
+     * @param bool $disableDirtyDetection
+     */
+    public static function disableDirtyDetection(bool $disableDirtyDetection)
```

Comment:
@weisswurstkanone since this sets the flag, it should be named accordingly. 
or maybe even better, we make 2 methods: `disableDirtyDetection()` and `enableDirtyDetection()`